### DISCUSSION TITLE
Wii u gamepad support? 😳

### DISCUSSION THREAD

**Discussion by RumpleStiltzchen** [ndeadly/MissionControl#491] [Ideas]
The screen and speakers wouldn't work as they probably use something like 802.11n but the actual buttons I am pretty sure use Bluetooth. I think it would be fun.

**Comment by ndeadly** [comment]
As far as I know the entire thing, buttons included, operates over some kind of wifi protocol. If you can show me proof of the button side of things operating via Bluetooth then I'll look into it, but I don't think that this is the case.

**Reply by RumpleStiltzchen** [reply]
I'm on it!

Not today as I'm busy but like in the near future

**Comment by Masamune3210** [comment]
Would also run into the issue that the only way to pair the Gamepad to anything involves WiiU-specific stuff unless you want to put in the effort in explaining to users how to figure out the code that the Gamepad is looking for

**Reply by RumpleStiltzchen** [reply]
Mm that is valid
